types/account.d.ts

```diff
@@ -256,6 +260,7 @@ export type UpdateCustomerConsentsArgs = {
   config: Config;
   customer: Customer;
   consents: CustomerConsent[];
+  consentValues: Record<string, string | boolean>;
```

Comment:
We need to try to make the cleeng and jwp services use 1 format, not just create a second property with similar but slightly different formatted data. Why can't we mutate consentValues into an array of `CustomerConsents`?